### PR TITLE
Update  for Sherlock 

### PR DESCRIPTION
**Pull request by matteofrigo5**
Main changes:
1. Change the  `INSTALL_DIR_ROOT`
2. Change the ` DOCKER_ROOT_IMAGE` (update to CUDA 11.7.1)

Relates to https://github.com/GEOS-DEV/GEOS/pull/2977

### CODE REVIEW

File: .github/workflows/docker_build_tpls.yml [hunk 1]
```diff
@@ -48,7 +48,7 @@ jobs:
             DOCKERFILE: docker/Stanford/Dockerfile
             DOCKER_ROOT_IMAGE: jafranc/sherlock-gcc10.1.0-openmpi4.1.2-cuda11.5.0-openblas0.3.10-zlib1.2.11-no-geosx:0.0.1
             HOST_CONFIG: docker/Stanford/sherlock-gcc10-ompi4.1.2-openblas0.3.10-cuda11.5.0-sm80.cmake
-            INSTALL_DIR_ROOT: /oak/stanford/schools/ees/COLLABORATIONS/geosx/GPU
+            INSTALL_DIR_ROOT: /home/groups/tchelepi/geosx-sherlock/GPU
```
Comment (CusiniM) [inline .github/workflows/docker_build_tpls.yml:51]: why not just `geos` ?

**Review by CusiniM — APPROVED** [review]

File: .github/workflows/docker_build_tpls.yml (hunk 1 above)
Comment (TotoGaz) [reply .github/workflows/docker_build_tpls.yml:51]: Yes, put `geos` whenever you can.

**Review by TotoGaz — APPROVED** [review]

**Review by TotoGaz — APPROVED** [review]
Please update the title and the description of the PR.

**Comment by jafranc** [comment]
@matteofrigo5 Are we sure on the final path it will be in ? Or will it be in `/oak/stanford/groups/tchelepi`

File: docker/Stanford/Sherlock.Dockerfile [hunk 2]
```diff
@@ -182,6 +182,6 @@ ENV CC=${SHERLOCK_GCC_INSTALL_DIR}/bin/gcc \
     MPIFC=${SHERLOCK_OPENMPI_INSTALL_DIR}/bin/mpifort \
     MPIEXEC=${SHERLOCK_OPENMPI_INSTALL_DIR}/bin/mpiexec \
 # An additional `LD_LIBRARY_PATH` action is needed for the tools to work.
-    LD_LIBRARY_PATH=${SHERLOCK_OPENMPI_INSTALL_DIR}/lib:${SHERLOCK_GCC_INSTALL_DIR}/lib64:${SHERLOCK_OPENBLAS_INSTALL_DIR}/lib:${SHERLOCK_ZLIB_INSTALL_DIR}/lib:${LD_LIBRARY_PATH}
+    LD_LIBRARY_PATH=${SHERLOCK_OPENMPI_INSTALL_DIR}/lib:${SHERLOCK_GCC_INSTALL_DIR}/lib64:${SHERLOCK_OPENBLAS_INSTALL_DIR}/lib:${SHERLOCK_CUDA_INSTALL_DIR}/lib64:${SHERLOCK_ZLIB_INSTALL_DIR}/lib:${LD_    LIBRARY_PATH}
```
Comment (TotoGaz) [inline docker/Stanford/Sherlock.Dockerfile:185]: 😨 fix those spaces!

**Comment by TotoGaz** [comment]
> Do I need to edit any other file?

I'd say it's OK!
Now it's possible (and probably a good idea 🤣 ) to test before merging!

**Comment by jafranc** [comment]
> > I've create two distinct directories, namely `/oak/stanford/groups/tchelepi/geos-sherlock/CPU` and `/oak/stanford/groups/tchelepi/geos-sherlock/GPU`, to segregate the binaries for the respective deployments of CPU and GPU. @jafranc do you think it might create some problems?
> 
> No that's good, then this PR should reflect those choices :) And you're good to merge and deploy
> 
> The PR reflect those choices:
> 
> https://github.com/GEOS-DEV/thirdPartyLibs/blob/23c14997f41c4c783bbdec27d349a13f23ea67a6/.github/workflows/docker_build_tpls.yml#L45
> 
> https://github.com/GEOS-DEV/thirdPartyLibs/blob/23c14997f41c4c783bbdec27d349a13f23ea67a6/.github/workflows/docker_build_tpls.yml#L51
> 
> Do I need to edit any other file?

Not that I recall, @TotoGaz is it the only location even post-GA introduction ?

**Comment by TotoGaz** [comment]
> is it the only location even post-GA introduction ?

The TPLs and `geos` will be installed in this folder.

**Pull request merged**